stock_picking_invoice_link/views/stock_view.xml

```diff
@@ -25,6 +29,10 @@
         <field name="name">stock_picking_invoice_link.stock.move.form</field>
         <field name="model">stock.move</field>
         <field name="inherit_id" ref="stock.view_move_form" />
+        <field
+            name="groups_id"
+            eval="[(6, 0, [ref('account.group_account_invoice')])]"
```

Comment:
There are multiple groups that gives access to invoices, so this is not true. Why this hasn't been a problem till now?